Engine/DataFeeds/SubscriptionDataReader.cs

```diff
@@ -202,6 +202,22 @@ object IEnumerator.Current
                 }
             }
 
+            // load up the map and factor files for underlying of equity option
+            if (!config.IsCustomData && config.SecurityType == SecurityType.Option)
```

Comment:
Now that there's two of these if blocks, I wonder if the config should expose something like, `IsMappable` or `RequiresSymbolMapping` to encapsulate the conditions... regarding the different behavior, perhaps the `MapFileResolver` should know what do to when given an options symbol (or maybe any derivative symbol)